features/step_definitions/fincap_pages_steps.rb

```diff
@@ -0,0 +1,67 @@
+Given(/^I have an insight page type$/) do
+  english_site = Comfy::Cms::Site.find_or_create_by(
+    label: 'en',
+    identifier: 'money-advice-service-en',
+    hostname: 'localhost:3000',
+    path: 'en',
+    locale: 'en',
+    is_mirrored: true
+  )
+
+  english_site.layouts.find_or_create_by(
+    identifier: 'insight',
+    label: 'Insight',
+    content:  <<-CONTENT
+      {{ cms:page:content:rich_text }}
+      {{ cms:page:overview }}
+      {{ cms:page:countries }}
+      {{ cms:page:links_to_research }}
+      {{ cms:page:contact_details }}
+      {{ cms:page:year_of_publication }}
+    CONTENT
+  )
+end
+
+Given(/^I am on the homepage$/) do
+  home_page.load
+end
+
+Given(/^I click on "([^"]*)"$/) do |button|
+  page.click_on(button)
```

Comment:
```ruby
click_on(button)
```

Should work just as well. It will not be in conflict with `new_page`. Once you've navigated to the right page calling the `page` object  itself is redundant.